src/store/middleware.js

```diff
@@ -15,6 +15,7 @@ import { updateKeyStatsAction } from "../features/key-stats";
 import { updateStockTickerAction } from "../features/stock-ticker";
 import { updateChartDataAction } from "../features/chart";
 import { resetAction } from "../App-actions";
+import { resetChartDataAction } from "../features/chart/redux/actions";
```

Comment:
This import should be collapsed with the one on line 16, as:
`import { resetChartDataAction, updateChartDataAction } from "../features/chart";`